### PR TITLE
Avoid importing astropy.stats for astropy.modeling

### PR DESCRIPTION
**Pull request by astrofrog**
This removes the only import of astropy.stats in astropy.modeling, which speeds up the import of astropy.modeling from ~1.8 seconds to ~1.5 seconds on my machine (as profiled with ``-X importtime``). I noticed this while debugging an issue in astropy-benchmarks which is caused by importing astropy.modeling ultimately importing pytest. This doesn't fix the root cause (see https://github.com/astropy/astropy/pull/7932 for that) but avoids a long chain of imports that isn't needed.

### CODE REVIEW

**Comment by astrofrog** [comment]
@mhvk - you mean to check the value of GAUSSIAN_SIGMA_TO_FWHM is the same in both modules?

**Comment by mhvk** [comment]
Yes, that's what I meant. Really shouldn't change, of course...

**Comment by astrofrog** [comment]
@mhvk - added!

File: astropy/modeling/functional_models.py [hunk 1]
```diff
@@ -25,6 +24,11 @@
 TWOPI = 2 * np.pi
 FLOAT_EPSILON = float(np.finfo(np.float32).tiny)
 
+# Note that we define this here rather than using the value defined in
+# astropy.stats to avoid importing astropy.stats every time astropy.modeling
+# is loaded.
+GAUSSIAN_SIGMA_TO_FWHM = 2.0 * np.sqrt(2.0 * np.log(2.0))
```
[the remark this answers is not in the document]
Comment (astrofrog) [reply astropy/modeling/functional_models.py]: *Practicality beats purity*

**Pull request merged**